### PR TITLE
Combine Users & Root artifacts into one

### PR DESCRIPTION
**Pull request by timevortex**
Creating one artifacts to access common shell configurations and history files for both system users and root.

[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/63)

### CODE REVIEW

File: definitions/linux.yaml [hunk 1]
```diff
@@ -94,8 +94,17 @@ sources:
 labels: [Configuration Files]
 supported_os: [Linux]
 ---
+name: AllShellConfigs
+doc: Common shell configuration files containing global, users & root settings.
+sources:
+- type: ARTIFACT
+  attributes:
+    names: [GlobalShellConfigs, UsersShellConfigs, RootUserShellConfigs]
```
Comment (joachimmetz) [inline definitions/linux.yaml]: Use long form:
names:
- GlobalShellConfigs
...

File: definitions/linux.yaml [hunk 2]
```diff
@@ -107,6 +116,15 @@ sources:
 labels: [Configuration Files]
 supported_os: [Linux]
 ---
+name: AllUsersShellHistory
+doc: Common shell history files for root and users.
+sources:
+- type: ARTIFACT
+  attributes:
+    names: [UsersShellConfigs, RootUserShellConfigs]
```
Comment (joachimmetz) [inline definitions/linux.yaml]: long form here as well

**Comment by timevortex** [comment]
Done! Please take a look.

**Comment by joachimmetz** [comment]
LGTM

**Pull request merged**